utils/src/field_helpers.rs

```diff
@@ -106,23 +100,25 @@ impl<F: Field> FieldHelpers<F> for F {
             bits
         })
     }
+}
 
-    fn from_big(big: BigUint) -> Result<F> {
-        let mut bytes = big.to_bytes_le();
+/// Field element helpers for [BigUint]
+pub trait BigFieldHelpers<F> {
+    /// Deserialize from big unsigned integer
+    fn from_big(big: BigUint) -> Result<F>;
 
-        // Pad with zeros until multiple of 32 if necessary because the BigUint to_bytes_le function gives the smallest possible vector of bytes
-        if bytes.len() % 32 != 0 {
-            // smallest larger multiple of 32 bytes
-            let goal = 32 + bytes.len() / 32;
-            bytes.extend(std::iter::repeat(0).take(goal - bytes.len()));
-        }
+    /// Serialize field element as a big unsigned integer
+    fn to_big(self) -> BigUint;
+}
 
-        F::deserialize(&mut &bytes[..]).map_err(|_| FieldHelpersError::DeserializeBytes)
+impl<F: PrimeField> BigFieldHelpers<F> for F {
+    fn from_big(big: BigUint) -> Result<F> {
+        big.try_into()
+            .map_err(|_| FieldHelpersError::DeserializeBytes)
     }
 
     fn to_big(self) -> BigUint {
-        let bytes = self.to_bytes();
-        BigUint::from_bytes_le(&bytes)
+        self.into_repr().into()
```

Comment:
I'm wondering why we need to wrap them in our own trait? If they're already in the PrimeField trait?

Reply:
(by that I mean, why not just use the PrimeField functions directly where needed?)

Reply:
I thought to_big() was easier to remember/more readable than the `into_repr().into()`, and similar for the from_big(). At least for now where I am using them in some functions of FFMul, it feels more immediate for me to use our wrapper. When the gates are finished though, we may realise that they are not being used that much and we may be able to get rid of them.

Reply:
you can just do `into()` wihtout the `into_repr()` though, similar with the other one

Reply:
for example:

```rust
let b: BigUint = Fp::from(3u32).into();
```

Reply:
yeah it worked! I removed the to_big but left from_big for now